src/lazy/expanded/compiler.rs

```diff
@@ -602,15 +712,11 @@ impl TemplateCompiler {
         let mut expressions = lazy_sexp.iter();
         // Convert the macro ID (name or address) into an address. If this refers to a macro that
         // doesn't exist yet, this will return an error. This prevents recursion.
-        // TODO: Consider storing the name of the invoked target macro in the host's definition
-        //       as debug information. The name cannot be stored directly on the
-        //       TemplateBodyMacroInvocation as that would prevent the type from being `Copy`.
-        let (_maybe_name, macro_address) =
-            Self::name_and_address_from_id_expr(tdl_context, expressions.next())?;
+        let macro_ref = Self::name_and_address_from_id_expr(tdl_context, expressions.next())?;
```

Comment:
🗺️ Now that we're using `Rc` to hold a reference to the `Macro` itself, the `name` is always available.